lib/msf/core/payload/macho.rb

```diff
@@ -53,6 +54,32 @@ def to_dylib(name)
     raw_data
   end
 
+  def sign
+    raw_data = @macho.serialize
+    code_signature_index = @macho[:LC_CODE_SIGNATURE][0].dataoff
+    code_signature = raw_data[code_signature_index..]
+    s_magic, s_length, s_count, code_indexes = code_signature.unpack("N3a*")
+    raise "Invalid Magic!" if s_magic != 0xfade0cc0
+    indexes = code_indexes.unpack("N#{s_count*2}a*")
+    code_directory = indexes.pop
+    magic, length, version, flags, hash_offset, ident_offset, n_special_slots, n_code_slots, code_limit, hash_size, hash_type, platform, page_size, spare2, hash_list = code_directory.unpack("N9C4Na*")
+    raise "Invalid Magic2!" if magic != 0xfade0c02
```

Comment:
Is the first magic number related to the code signature? If so, could these raise messages be changed to something like `Invalid magic number for macho code signature` and `Invalid magic number for macho` or something along those lines to be a little more descriptive?